proto/hummock.proto

```diff
@@ -10,7 +10,8 @@ message SstableInfo {
   uint64 id = 1;
   KeyRange key_range = 2;
   uint64 file_size = 3;
-  repeated uint32 table_ids = 4;
+  string path_prefix = 4;
+  repeated uint32 table_ids = 5;
```

Comment:
When adding a new field in proto, we should append it to the end with a larger id to ensure backward compatibility.